packages/ses/src/main.js

```diff
@@ -12,5 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export { lockdown } from './lockdown-shim.js';
-export { Compartment, ModuleStaticRecord } from './compartment-shim.js';
+import { lockdown, harden } from './lockdown-shim.js';
+import { Compartment, ModuleStaticRecord } from './compartment-shim.js';
+
+Object.assign(globalThis, {
+  lockdown,
+  harden,
+  Compartment,
+  ModuleStaticRecord,
```

Comment:
All shimmed pervasively global names must be added to the appropriate whitelists. I don't see `lockdown` and `ModuleStaticRecord` (or `StaticModuleRecord`) in `intrinsic-names.js`

I do see `ModuleStaticRecord` in `intrinsic-globals.js`, but I don't see `lockdown` there.

`lockdown` missing from `whitelist.js` as well.

We should have a test that fails on such omission, because it is an easy mistake to keep making. 

TODO(markm): the whole intrinsic management system now consists of nine separate files with too much redundancy, e.g., between `intrinsic-names.js`, `intrinsic-globals.js`, and `whitelist.js`. We should simplify, make it easier to maintain, and self-test consistency among what redundancy remains. Also, `check-intrinsics.js` seems vacuous. Bad edit from a previously meaningful state?

Reply:
Filed TODO as https://github.com/Agoric/SES-shim/issues/317

Reply:
This is the kind of thing that I worry most about. Thanks for catching this.

Reply:
Adding `lockdown` to the whitelist caused tests to fail, so I’ll come back to it when we have a firm resolution as to whether it should preserve itself after `lockdown`.

Reply:
ok. But make sure the state as of merging this PR is consistent with itself.